ansible/group_vars/all/system.yaml

```diff
@@ -32,7 +32,8 @@ docker_deb_gpg_key_url: "https://packages.d2iq.com/konvoy/stable/linux/{{ ansibl
 docker_deb_release_name: "{{ ansible_distribution_release }}"
 docker_deb_component: "main"
 
-kubernetes_full_version: "{{ kubernetes_version }}{{ kubernetes_build_meta }}"
+kubernetes_full_version: "{{ kubernetes_version }}{{ kubernetes_build_metadata }}"
```

Comment:
`kubernetes_build_metadata=fips.0`

How does `kubernetes_full_version` end up with `mesosphere/kube-apiserver:v1.21.6+fips.0`, wouldn't it be `mesosphere/kube-apiserver:v1.21.6fips.0`?

Reply:
Good catch.. I was actually curious how it was being set because the variable was wrong previously. It's being overwritten in the code that is generating the version internally.